src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -99,7 +99,8 @@ class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLog
         .map(id => getAudience(id.getValue)).collectResults
       ddm <- DDM(emd, audiences)
       fedoraIDs <- fedoraProvider.getSubordinates(datasetId)
-      _ <- simpleChecker.isSimple(emd, ddm, amd, startWith("easy-jumpoff:", in = fedoraIDs))
+      _ <- if (strict) simpleChecker.isSimple(emd, ddm, amd, startWith("easy-jumpoff:", in = fedoraIDs))
+           else Success(()) // TODO perhaps log a warning if not strict?
```

Comment:
@lindareijnhoudt 

The CSV output has a column `transformationType` which is always the same in one run. Should we mark violations somehow in this field?

Reply:
i suggest to put these warnings also in the `comments` column

Reply:
New implementation: compare 
https://github.com/DANS-KNAW/easy-fedora2vault/blob/5c23d996bbcac1640f3bb77e0802e22d971e0867/src/test/scala/nl/knaw/dans/easy/fedora2vault/AppSpec.scala#L143
with
https://github.com/DANS-KNAW/easy-fedora2vault/blob/5c23d996bbcac1640f3bb77e0802e22d971e0867/src/test/scala/nl/knaw/dans/easy/fedora2vault/AppSpec.scala#L97